### ISSUE TITLE
internationalization support

### ISSUE THREAD

**Issue by jolan** [decred/dcrstakepool#12]
feature request by @girino

Looks like this would be easy to implement:

https://github.com/nicksnyder/go-i18n

**Comment by mverrilli** [comment]
Yep I have this done already, but I'll need to rebase it and I'm a little short on time at the moment.
